cudf/io.py

```diff
@@ -106,6 +108,9 @@ def read_csv(filepath, lineterminator='\n',
     dtype_ptr = ffi.new('char*[]', arr_dtypes)
     csv_reader.dtype = dtype_ptr
 
+    if sep is not None:
+        delimiter = sep
```

Comment:
Is this the correct behavior? What if both `delimiter` and `sep` are defined? Should an exception be thrown or does one override the other by default?